docs/api/qiskit/0.24/qiskit.aqua.algorithms.AmplitudeEstimation.mdx

```diff
@@ -93,7 +93,7 @@ $$
 
   ### a\_factory
 
-  <Function id="qiskit.aqua.algorithms.AmplitudeEstimation.a_factory" signature="property a_factory">
+  <Function id="qiskit.aqua.algorithms.AmplitudeEstimation.a_factory" signature="a_factory" modifiers="property">
```

Comment:
Hm, I didn't realize we have properties marked as functions. These should probably be moved to Attributes. https://github.com/Qiskit/documentation/issues/1352